fleet/lib/rook-ceph-conf/charts/manke/templates/cm-cephcli.yaml

```diff
@@ -10,34 +10,6 @@ data:
     ceph orch set backend ""
     ceph mgr module disable rook
 
-    waitfornfs jhome
-    ceph nfs export rm jhome /jhome
-    ceph nfs export create cephfs jhome /jhome jhome /volumes/_nogroup/jhome/edbad2c0-fd61-4cef-abcc-612656dd0acd
-
-    waitfornfs lsstdata
-    ceph nfs export rm lsstdata /lsstdata
-    ceph nfs export create cephfs lsstdata /lsstdata lsstdata /volumes/_nogroup/lsstdata/9a3a6099-59c3-4d9d-b43d-c8b85c213382
-
-    waitfornfs project
-    ceph nfs export rm project /project
-    ceph nfs export create cephfs project /project project /volumes/_nogroup/project/bee6346d-ea45-421b-9293-425fc7a0a3b3
-
-    waitfornfs scratch
-    ceph nfs export rm scratch /scratch
-    ceph nfs export create cephfs scratch /scratch scratch /volumes/_nogroup/scratch/d4551663-82aa-482c-8bb1-2d5561cca50d
-
-    waitfornfs obs-env
-    ceph nfs export rm obs-env /obs-env
-    ceph nfs export create cephfs obs-env /obs-env obs-env
-
-    waitfornfs auxtel
-    ceph nfs export rm auxtel /auxtel
-    ceph nfs export create cephfs auxtel /auxtel auxtel
-
-    waitfornfs comcam
-    ceph nfs export rm comcam /comcam
-    ceph nfs export create cephfs comcam /comcam comcam
-
     ceph mgr module enable rook
     ceph orch set backend rook
```

Comment:
We want to leave the orch disabled now. See elqui/pillan as a reference.

Reply:
left them as the one in elqui 👍🏻